src/render/tree/mod.rs

```diff
@@ -199,74 +198,74 @@ impl Display for Tree {
         let root = self.root();
         let level = self.level();
         let theme = ui::get_theme();
-        let prune = self.context().prune;
-        let mut output = String::from("");
-
-        #[inline]
-        fn extend_output(output: &mut String, node: &Node, prefix: &str) {
-            output.push_str(&format!("{prefix}{node}\n"));
+        let size_left = self.context().size_left;
+
+        fn extend_output(
```

Comment:
Would these not benefit from being `#[inline]`?

Reply:
...maybe. `#[inline]` is, at best, a [suggestion](https://nnethercote.github.io/perf-book/inlining.html) -- this function is so tiny it's almost guaranteed to be inclined (in release builds, at least).

Even if not inlined without the attribute (but inlined with it), I don't think you'd be able to see a measurable performance change -- the variance of the io-bound traversal would probably dwarf the change.

Reply:
that is fair